modules/ccs-gcp-customer-procedure.adoc

```diff
@@ -9,7 +9,13 @@
 The Customer Cloud Subscription (CCS) model allows Red{nbsp}Hat to deploy and manage {product-title} into a customer's Google Cloud Platform (GCP) project. Red Hat requires several prerequisites to provide these services.
 [NOTE]
 ====
-The following requirements in this topic apply to {product-title} on {GCP} clusters created using both the service account and Workload Identity Federation authentication type. For additional requirements that apply to the service account authentication type only, see _Service account authentication type procedure_. For additional requirements that apply to the Workload Identity Federation authentication type only, see _Workload Identity Federation authentication type procedure_.
+The following requirements in this topic apply to {product-title} on {GCP} clusters created using both the Workload Identity Federation (WIF) and service account authentication type.
+Red{nbsp}Hat recommends using WIF as the authentication type for installing and interacting with the OpenShift Dedicated cluster deployed on GCP because it provides enhanced security.
```

Comment:
```suggestion
Red{nbsp}Hat recommends using WIF as the authentication type for installing and interacting with an {product-title} cluster deployed on GCP because WIF provides enhanced security.
```